README.md

```diff
@@ -84,9 +84,9 @@ Card / Agency | Location | Notes
 City Union | :cn: Mainland China | :new:
 [Clipper][clipper] | :us: San Francisco, CA, USA
 [Compass][compass] | :canada: Vancouver, BC, Canada | :new: :one:
-[Crimea Trolleybus][crimea] | Crimea | :new: :closed_lock_with_key: `MFC`
+[Crimea Trolleybus][crimea] | Crimea, Eastern Europe | :new: :closed_lock_with_key: `MFC`
 [Cubic Nextfare][nextfare] | :earth_americas: _many locations_ | :new: :closed_lock_with_key: `MFC`
-[EasyCard][easycard] | :taiwan: Taipei, Taiwan | :closed_lock_with_key: `MFC`
+[EasyCard][easycard] | :taiwan: Taipei, East Asia | :closed_lock_with_key: `MFC`
```

Comment:
"Taiwan" needs to be removed altogether. Feel free to adjust the patch or I'll propose it as a separate patch.

Reply:
No more Taiwan.